hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSImageFormatProtobuf.java

```diff
@@ -172,13 +172,55 @@ public LoaderContext getLoaderContext() {
       return ctx;
     }
 
+    private class DigestThread extends Thread {
```

Comment:
Javadoc mentioning that this a thread for parallel MD5 computing to increase performance when loading.